tests/sentry/sentry_metrics/test_all_indexers.py

```diff
@@ -44,7 +44,7 @@ def indexer_cls(request):
     return request.param
 
 
-@pytest.fixture(params=USE_CASE_KEYS)
+@pytest.fixture(params=UseCaseKey)
```

Comment:
Nit: I'm happy with using the enum itself as an iterable, but we should get rid of the `USE_CASE_KEYS` in L27